spec/ics-027-interchain-account/README.md

```diff
@@ -0,0 +1,285 @@
+---
+ics: 27  
+title: Interchain Account  
+stage: Draft  
+category: IBC/TAO  
+author: Tony Yun <yunjh1994@everett.zone>, Dogemos <josh@tendermint.com>  
+created: 2019-08-01  
+modified: 2019-11-08  
+---  
+
+## **Synopsis**  
+
+This standard document specifies packet data structure, state machine handling logic, and encoding details for the account management system over an IBC channel between separate chains.  
+
+### **Motivation**  
+
+On Ethereum, there are two types of accounts: externally owned accounts, controlled by private keys, and contract accounts, controlled by their contract code [[ref](https://github.com/ethereum/wiki/wiki/White-Paper)]. Similar to Ethereum's CA(contract accounts), Interchain accounts are managed by another chain while retaining all the capabilities of a normal account (i.e. stake, send, vote, etc). While an Ethereum CA's contract logic is performed within Ethereum's EVM, Interchain accounts are managed by another chain via IBC in a trustless way.  
```

Comment:
```suggestion
On Ethereum, there are two types of accounts: externally owned accounts, controlled by private keys, and contract accounts, controlled by their contract code [[ref](https://github.com/ethereum/wiki/wiki/White-Paper)]. Similar to Ethereum's CA (contract accounts), interchain accounts are managed by another chain while retaining all the capabilities of a normal account (i.e. stake, send, vote, etc). While an Ethereum CA's contract logic is performed within Ethereum's EVM, interchain accounts are managed by another chain via IBC in a trustless way.  
```

Reply:
Unfortunately, I think 'trustless' is so overutilised as to render the term meaningless - can we state something more specific, e.g. 'in a way such that the owner of the account retains full control over how it behaves'?